placenames.mss

```diff
@@ -4,34 +4,22 @@
 @state-labels: desaturate(darken(@admin-boundaries, 5%), 20%);
 
 .country {
-  [zoom >= 3][way_pixels > 1000][way_pixels < 360000] {
+  [zoom >= 3][way_pixels > 1000][way_pixels < 192000] {
```

Comment:
This change would require changes to the way_pixels limit in `#admin-text` so that they match for; currently ` [admin_level = '1'][way_pixels >= 360000], [admin_level = '2'][way_pixels >= 360000],` would need to change to [way_pixels >= 192000]